parquet/write.go

```diff
@@ -64,8 +68,29 @@ func convertSchema(sch *arrow.Schema) *arrow.Schema {
 	return newSchema
 }
 
-// castExtensionColsToString casts extension columns to string. It does not release the original record.
-func castExtensionColsToString(rec arrow.Record) (arrow.Record, error) {
+func isUnsupportedType(t arrow.DataType) bool {
+	switch dt := t.(type) {
+	case *arrow.DayTimeIntervalType, *arrow.DurationType, *arrow.MonthDayNanoIntervalType, *arrow.MonthIntervalType: // unsupported in pqarrow
+		return true
+	case *arrow.LargeBinaryType, *arrow.LargeListType, *arrow.LargeStringType: // not yet implemented in arrow
```

Comment:
Ideally we should fall back to the non-large types (and zero copy contents?) in convertschema and other places, but this was easier and less messy for now.